.github/workflows/tag_image_push_rhel.yml

```diff
@@ -70,21 +71,6 @@ jobs:
           fi
           echo "RELEASE_VERSION=${RELEASE_VERSION}" >> $GITHUB_ENV
 
-      - name: Check HZ major version
-        run: |
-          HZ_MAJOR_VERSION=$(echo "${HZ_VERSION:0:1}")
-          if [[ "$HZ_MAJOR_VERSION" != "$REQUIRED_HZ_MAJOR_VERSION" ]]; then
-            echo "Major version must be ${REQUIRED_HZ_MAJOR_VERSION} but detected: ${HZ_MAJOR_VERSION}"
-            exit 1
-          fi
-          echo "HZ_MAJOR_VERSION=${HZ_MAJOR_VERSION}" >> $GITHUB_ENV
```

Comment:
We need HZ_MAJOR_VERSION

Reply:
> We need HZ_MAJOR_VERSION

d2a8fb81b38d8a1c14ecfc05b35a541402d8ce01

Made it dynamic as otherwise it'll break with v6.